docs/releases/1.10.txt

```diff
@@ -209,7 +209,9 @@ Tests
 URLs
 ^^^^
 
-* ...
+* Resolution of URLs outside of request contexts (management commands,
```

Comment:
An addition in django.setup() allows URL resolving that happens outside of the request/response cycle (e.g. in management commands and standalone scripts) to take FORCE_SCRIPT_NAME into account when it is set.